woocommerce/woocommerce-gateway-stripe
# Fixed Stripe Link styles and image files missing

## Changes proposed in this Pull Request:

Fixes the issue where executing the `npm run build` command deletes the `assets/css/striple-link.css` file and breaks the Stripe Link trigger button.

## Testing instructions
- Execute `npm run build`
- Verify the file `assets/css/stripe-link.css` exist after the build process.
- Verify the Stripe Link trigger button styles are correctly loaded.
- Verify the Stripe Link logo button is displayed.

---


**Post merge**
